gems/sorbet-runtime/lib/types/types/attached_class.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+# typed: true
+
+module T::Types
+  # Modeling AttachedClass properly at runtime would require additional
+  # tracking, so at runtime we permit all values and rely on the static checker.
```

Comment:
Probably worth pointing out that this is largely consistent with all other type members.